dmcontent/questions.py

```diff
@@ -667,9 +667,9 @@ def filter_value(self):
         value = self._service_data.get(self.id, '')
         if options and value:
             for option in options:
-                if ('filter_label' in option and 'value' in option and (
-                            option.get('value') or option.get('label')) == value):
-                    return option['filter_label']
+                if 'filter_label' in option and 'value' in option:
```

Comment:
Splitting up the conditional and nesting `ifs` here as a fix for "continuation line over-indented for hanging indent" doesn't feel like strictly a Flake8 fix. You could just have deleted some spaces for that.
Personally I prefer the single `if` as it was, but it's marginal and either works I guess ¯\\_(ツ)_/¯

Reply:
The single if will look like this otherwise 
```
        if options and value:
            for option in options:
                if (
                    'filter_label' in option 
                    and 'value' in option
                    and (option.get('value') or option.get('label')) == value
                ):
                        return option['filter_label']
```

Also I think splitting out the separate actions (ie 'checking the keys are in the dict' and 'checking the values of keys in the dict') creates a nice context boundary. 

For me it reads like:

if key one and key two are there
    if value two or value three match `value`
        do the thing

I like how i can disregard the first if statement while i'm working out what the second bit does (which feels a bit gnarly to me).

Something like this would be nicer 
`if 'filter_label' in option and 'value' in option and value in [option.get('value'), option.get('label')]:`

But that is 122 characters.

Reply:
Nailed it.

Reply:
Yep, nailed it.